### PR TITLE
feat: skip updatecli stage if there is no updatecli folder

### PR DESCRIPTION
**Pull request by lemeurherve**
Tested here: https://github.com/jenkins-infra/docker-plugin-site-issues/pull/4

### CODE REVIEW

File: vars/updatecli.groovy [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import java.util.*
```
Comment (timja) [inline vars/updatecli.groovy:1]: I don't think this is what you meant to import

Comment (lemeurherve) [reply vars/updatecli.groovy]: Indeed ^^

**Review by timja — APPROVED** [review]

**Comment by dduportal** [comment]
Testing the case with updatecli manifest present in this draft PR: https://github.com/jenkins-infra/docker-aws/pull/5

**Review by dduportal — APPROVED** [review]
Great job !!

**Pull request merged**